src/main/java/org/jitsi/jicofo/jigasi/JigasiDetector.java

```diff
@@ -58,6 +58,10 @@ public JigasiDetector(
             breweryName,
             ColibriStatsExtension.ELEMENT_NAME,
             ColibriStatsExtension.NAMESPACE);
+
+        // We prefer to communicate with jitsi-videobridge without going
+        // through the MUC.
+        setUseOccupantJid(false);
```

Comment:
so the intention here is to discover the jvb's (non-muc) jid through the muc, and then use that to communicate with it as opposed to its muc jid? do i have that right?  i assume this is a scheme we'll use with the other services who register like this as well?

Reply:
> do i have that right?

Yes. Sending IQs through the MUC is just extra complexity because it terminates them (and you can't correlate the IDs anymore).

> i assume this is a scheme we'll use with the other services who register like this as well?

I think it makes sense to use it for jibri and jigasi as well, but I only did it for the bridge because I wasn't sure whether they would work with the current code.